pkg/sql/backfill.go

```diff
@@ -209,6 +221,39 @@ func (sc *SchemaChanger) runBackfill(
 		}
 	}
 
+	// Make check constraints public (or drop them, if it's a rollback).
+	if len(addedChecks) > 0 || len(droppedChecks) > 0 {
+		_, err := sc.leaseMgr.Publish(ctx, sc.tableID,
+			func(desc *sqlbase.MutableTableDescriptor) error {
+				checks := make([]*sqlbase.TableDescriptor_CheckConstraint, 0)
+				for _, c := range desc.Checks {
+					found := false
+					for _, dropped := range droppedChecks {
+						if dropped.Name == c.Name {
+							found = true
+							break
+						}
+					}
+					if !found {
+						checks = append(checks, c)
+					}
+				}
+				for i := range addedChecks {
+					checks = append(checks, &addedChecks[i])
+				}
+
+				desc.Checks = checks
+				return nil
+			},
+			func(txn *client.Txn) error {
+				return nil
+			},
+		)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
as discussed in person: do we need to explicitly wait for one version here?

Reply:
Yes, done.